test/inductor/test_custom_lowering.py

```diff
@@ -137,6 +137,17 @@ def fn(inp, offsets, max_seq_len):
             fn(inp, offsets, max_seq_len), fn_opt(inp, offsets, max_seq_len)
         )
 
+    def test_unfold_zero_dimension_tensor(self):
+        def forward(x):
+            return torch.unfold_copy(dimension=1, input=x,size=0,step=7)
+
+        x = torch.rand([1,0], dtype=torch.float32)
+
+        y = forward(x)
+        compiled_y = torch.compile(forward, mode='max-autotune',fullgraph=True)(x)
```

Comment:
This test doesn't need max-autotune. Also, it should go in `test_torchinductor.py`

Reply:
Appreciate the quick review! I put it under CommonTemplate but I'm having trouble running it locally. 

I'm on a mac and it's looking like this line
`if HAS_CPU and not torch.backends.mps.is_available():`
is short circuiting the CommonTemplate tests from running. 

If I comment out the `and not torch.backends.mps.is_available()` part
`python test/inductor/test_torch
inductor.py -k test_unfold_zero_dimension_tensor`
passes!